### PR TITLE
LPS-147430 update dev dependencies to use dart sass

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/package.json [hunk 1]
```diff
@@ -16,8 +16,9 @@
 		"zone.js": "0.8.12"
 	},
 	"devDependencies": {
-		"@angular/cli": "^1.6.7",
-		"@angular/compiler-cli": "^5.0.0",
+		"@angular-devkit/build-angular": "^12.2.16",
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/package.json:19]: @izaera, Are these dependencies upgrade safe for our templates? Functionally the portlets build, but we wanted to confirm with you regarding the risks of such a change. Thanks!

Comment (izaera) [reply modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/package.json:19]: As long as they don't break the portlet, they should be safe.

Comment (simonjhy) [reply modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/package.json:19]: @izaera Thank your help.

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

**Pull request closed without merging**